bazel/utils/diff_test.bzl

```diff
@@ -22,28 +77,21 @@ def _diff_test_impl(ctx):
         if [[ ! -e "${{exp}}" ]]; then echo "Missing file: ${{exp}}"; exit 1; fi
         diff -u "${{exp}}" "${{act}}" ; RC=$?
         while [ "${{1:-}}" != "" ]; do
-          if [[ "$1" == "--update_goldens" ]]; then
```

Comment:
Can we revert this change, and instead I'll push a PR that will backport this into internal?  Especially for external users, the improved error messages are useful.

Reply:
done.